manifests/init.pp

```diff
@@ -1,3 +1,11 @@
-class helloworld {
+class helloworld (
+  $file_path = undef
+){
   notify { "Hello world!": message => "I am in the ${environment} environment"}
+  unless $file_path == undef and !("" in [$x]){
```

Comment:
Its copy pasta that I found in a message board answer from Henrick.  I think I can remove it as I think it was intended to also skip if the string was empty.